contracts/src/v0.8/automation/HeartbeatRequester.sol

```diff
@@ -32,7 +32,7 @@ contract HeartbeatRequester is TypeAndVersionInterface, ConfirmedOwner {
    * - HeartbeatRequester 1.0.0: The requester fetches the latest aggregator address from proxy, and request a new round
    *                             using the aggregator address.
    */
-  string public constant override typeAndVersion = "HeartbeatRequester 1.0.0";
```

Comment:
This is a place where we ignore the warning and keep the old name to prevent having to create a new function 

  // solhint-disable-next-line chainlink-solidity/all-caps-constant-storage-variables

Reply:
Aha, thank you so much Rens! I have been dealing with this one for the past hour. It is causing a bunch of cascading issues. I will update the PR. In the meantime, I update the description, explaining the 3 remaining warnings, please take a look. Thanks~